cl_dll/view.cpp

```diff
@@ -409,6 +411,7 @@ void V_CalcViewRoll ( struct ref_params_s *pparams )
 	viewentity = gEngfuncs.GetEntityByIndex( pparams->viewentity );
 	if ( !viewentity )
 		return;
+	pparams->viewangles[ROLL] = V_CalcRoll (pparams->viewangles, pparams->simvel, cl_viewrollangle->value, cl_viewrollspeed->value ) * 4;
```

Comment:
Should this be a += so as to not override the roll angle?

Reply:
I've actually never tried that before.. will do now.

Reply:
Adding `+=` made no in-game difference

Reply:
Probably because there was no viewangle roll. I suggest then changing it to += because the code right below also does += and not =.